.gitignore

```diff
@@ -42,8 +42,8 @@ derby.log
 ldap
 */dependency-reduced-pom.xml
 */*/dependency-reduced-pom.xml
-metrics/report.json
-metrics/.report.json.crc
+report.json
```

Comment:
please check the scala-test plugin settings in pom.xml, you should be able to skip these changes